Datastore/src/EntityMapper.php

```diff
@@ -328,6 +356,18 @@ public function valueObject($value, $exclude = false, $meaning = null)
                 break;
 
             case 'double':
+                // The mappings happen automatically for grpc hence
+                // this is required only incase of rest as grpc
+                // doesn't recognises `'Infinity'`, `'-Infinity'` and `'NaN'`.
```

Comment:
No need to backticks as this won't be parsed by IDEs

Reply:
Updated